### PR TITLE
[WIP] Feature/swagger version numbers 

### PR DESCRIPTION
**Pull request by janfriedli**
Is part of EVO403

### CODE REVIEW

**Comment by hairmare** [comment]
Needs a link to an example (like on graviton.nova.scapp.io) as well as to a schema.

**Comment by hairmare** [comment]
This still needs linking to an example as per my last comment.

Also, it is not really documenting the API. 

Please split up the contents that document the inner working of the feature into a guide type document and add some more information on the API parts to this article.

The API parts I want to see documented are what to expect from the API response as well as some documentation on the header that gets returned.

Please also add an explanation for self from an api users point of view.

**Comment by janfriedli** [comment]
https://github.com/libgraviton/graviton/tree/feature/core-bundle-readme is part of this

File: source/_pages/api/versions.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    }
+
+}
+```
+
+A working example can be found [here](https://graviton.nova.scapp.io/core/version).
+
+And the schema [here](https://graviton.nova.scapp.io/schema/core/version).
+
+## How to configure which version are reported
```
Comment (hairmare) [inline source/_pages/api/versions.md]: All of this section except the *self* explanation should go to https://github.com/libgraviton/graviton/pull/294

**Pull request merged**